cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,70 @@
 /// <reference types='cypress' />
+import HomeAndCataloguePageObject from "../support/pages/homeСatalogue.pageObject"
+import PageObject from "../support/PageObject";
+import PlaceOrderForm from "../support/pages/placeOrderForm.pageObject";
+
+
+let user
+
+const checkOut = new HomeAndCataloguePageObject();
+const pagObj = new PageObject();
+const orderPage = new PlaceOrderForm();
+
 
 describe('', () => {
-  before(() => {
+
+  beforeEach(() => { 
+    cy.task('createPlaceOrderUser').then(createPlaceOrderUser => {
+      user = createPlaceOrderUser;
+   });
 
   });
 
-  it('', () => {
+  it('should provide an ability to add product to the cart', () => {
+
+    checkOut.visit();    
+    checkOut.clickOnLink();
```

Comment:
these methods are super hardcoded, fix them
and this is not check out page - it's a home or product page